tests/posix/common/src/pthread.c

```diff
@@ -143,6 +143,13 @@ static void *thread_top_exec(void *p1)
 	return NULL;
 }
 
+static void *timedjoin_thread(void *p1)
+{
+	int sleep_duration_ms = POINTER_TO_INT(p1);
+	usleep(USEC_PER_MSEC * sleep_duration_ms);
```

Comment:
@ClaCodes - this is causing the compliance failure.

```suggestion

	usleep(USEC_PER_MSEC * sleep_duration_ms);
```

Reply:
Thanks, I added the newline.